Gemfile

```diff
@@ -42,7 +42,7 @@ group :test do
 end
 
 group :acceptance do
-  gem 'beaker', '~> 2.2'
+  gem "beaker", :git => 'git://github.com/puppetlabs/beaker.git', :ref => '7d0cccf22cc7289113de862fe4a4494872f574bb'
```

Comment:
@kbarber this is the beaker version that is pinned for compatibility with aio at the moment. As soon as a version of beaker that supports aio is released this pin should be replaced by a twiddle-wakka based on the release version.